app/scripts/modules/plugins/webpack.config.js

```diff
@@ -0,0 +1,114 @@
+'use strict';
```

Comment:
when does this script run? eg since plugins are loaded at runtime

Reply:
runs at build time of deck

Reply:
what exactly is it doing / what is it for? My understanding is webpack is for compiling (maybe transpiling too), since plugins aren't available at build time, I'm not sure why we have this file

Reply:
This is used to actually compile the module into a resulting library that can be consumed downstream by plugin developers

Reply:
hmmm I see you have some frontend portion of the stage-plugin, does that pull in this library for testing? or how does that work?